### DISCUSSION TITLE
What modules are required to run py3-validate-email?

### DISCUSSION THREAD

**Discussion by iSkylark** [karolyi/py3-validate-email#64] [Q&A] [answered]
Hi! 

I just installed py3-validate-email and when I try to run the test command I encounter an error:
"ModuleNotFoundError: No module named 'dns'"

Which packages and modules are required for it to work properly?

**Comment by reinhard-mueller** [comment, marked as answer]
The requirements are:
* dnspython (2.0)
* idna (2.10)
* 'filelock (3.0)

Actually, on proper install, the dependencies should be checked and, if missing, automatically installed. How did you install py3-validate-email?

**Reply by iSkylark** [reply]
pip3 install py3-valide-email

**Reply by reinhard-mueller** [reply]
That clearly should have installed dnspython as well. Can you check whether that's installed for you, and if yes, why it's not found?
